packages/batch-submitter/.lintstagedrc.yml

```diff
@@ -0,0 +1,6 @@
+src/**/*.ts:
```

Comment:
This is cool, although it does add a bit more mental overhead for files I need to change if I add a new folder or whatever. Any chance this can be generated automatically or that we can avoid needing this file altogether?

Reply:
I'm not aware of any way to automate this in general. We could maybe hook into the files defined in `tsconfig.json` but that would ignore `.js` files. Also `eslint` is generalized and could run on any kind of file type, so doing it "in general" may not be possible

Reply:
Is there a way to say `!node_modules` and `**/*.ts` and `**/*.js`?

Reply:
df8785b4